hermes-api/src/main/java/pl/allegro/tech/hermes/api/endpoints/OfflineRetransmissionEndpoint.java

```diff
@@ -0,0 +1,30 @@
+package pl.allegro.tech.hermes.api.endpoints;
+
+import java.util.List;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.Response;
+import pl.allegro.tech.hermes.api.OfflineRetransmissionRequest;
+import pl.allegro.tech.hermes.api.OfflineRetransmissionTask;
+
+import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
+
+@Path("offline-retransmission/tasks")
+public interface OfflineRetransmissionEndpoint {
+    @POST
+    @Consumes(APPLICATION_JSON)
+    Response createRetransmissionTask(OfflineRetransmissionRequest request);
+
+    @GET
+    @Produces(APPLICATION_JSON)
+    List<OfflineRetransmissionTask> getAllTasks();
```

Comment:
What do You think about packing this list of tasks into an object? In the future it will be much easier to add other fields if needed.

Reply:
I get what you mean but I followed the same rules we're actually returning list of objects, e.g topics, subscriptions etc. I think there is enough flexibility in the `OfflineRetransmissionTask` itself ;)